upstart/security-hq.conf

```diff
@@ -0,0 +1,32 @@
+# upstart script for security-hq
+
+start on runlevel [2345]
+stop on runlevel [016]
+
+setuid securityhq
+chdir /security-hq
+
+script
+  IF_64_BIT_OPTION=""
+  if $(java -version 2>&1 | grep -q 64-Bit);
+  then
+    IF_64_BIT_OPTIONS="-J-XX:+UseCompressedOops -J-XX:+UseConcMarkSweepGC"
+  fi
+
+  # Incremental mode if we are on 1 or 2 CPUs http://www.oracle.com/technetwork/java/javase/gc-tuning-6-140523.html#icms
+  INCREMENTAL_MODE=""
+
+  if [ $(nproc) -lt 3 ]
+  then
+    INCREMENTAL_MODE="-J-XX:+CMSIncrementalMode";
+  fi
+
+  TOTAL_MEMORY=$(grep MemTotal /proc/meminfo | awk '{ print $2 }')
+  HEAP_SIZE_IN_MB=$(perl -e "print int($TOTAL_MEMORY * 0.75 / 1024)")
+  HEAP_SIZE_CMD="-J-Xmx${HEAP_SIZE_IN_MB}m"
+
+  COMMAND="/security-hq/bin/security-hq $HEAP_SIZE_CMD $IF_64_BIT_OPTIONS $INCREMENTAL_MODE -J-XX:+PrintGCDetails -J-XX:+PrintGCDateStamps -J-Xloggc:gc.log -Dconfig.file=/security-hq/security-hq.conf"
+
+  echo "$COMMAND" >/security-hq/cmd.txt
+  $COMMAND >/security-hq/stdout.log 2>&1
```

Comment:
can we have logs under `/security-hq/logs` folder so that they stay in the same place with the application log.